setup/flavors/compose/mailu.env

```diff
@@ -29,7 +29,7 @@ POSTMASTER={{ postmaster }}
 # Choose how secure connections will behave (value: letsencrypt, cert, notls, mail, mail-letsencrypt)
 TLS_FLAVOR={{ tls_flavor }}
 
-# Authentication rate limit per IP (per /24 on ipv4 and /48 on ipv6)
+# Authentication rate limit per IP for failed login attempts or non-existing accounts (per /24 on ipv4 and /48 on ipv6)
```

Comment:
This does not accurately reflect what's currently in master; it may if/once #2772 is merged.

Reply:
@nextgens 
#2772 has been merged now. I think this is correct now?

The text itself contains a typo
```suggestion
# Authentication rate limit per IP for failed login attempts on non-existing accounts (per /24 on ipv4 and /48 on ipv6)
```